chromadb/server/fastapi/__init__.py

```diff
@@ -324,10 +339,18 @@ def version(self) -> str:
             ),
         ),
     )
-    def create_database(
-        self, database: CreateDatabase, tenant: str = DEFAULT_TENANT
+    async def create_database(
+        self, request: Request, tenant: str = DEFAULT_TENANT
     ) -> None:
-        return self._api.create_database(database.name, tenant)
+        def process_create_tenant(raw_body: bytes) -> None:
+            create = CreateDatabase.model_validate(orjson.loads(raw_body))
+            return self._api.create_database(create.name, tenant)
+
+        return await to_thread.run_sync(  # type: ignore
```

Comment:
is there a way to do tihs without `type: ignore`? I'm on a quest to get all our types correct

Reply:
The problem is that run_sync requires a specific interface, and mypy complains about the typing. I can remove the comment and let mypy complain :).

Reply:
What I meant is: Can we use it in a way which makes mypy happy?

Reply:
mypy is happy with casting :)